Sources/StreamChat/ChatClient.swift

```diff
@@ -79,12 +79,14 @@ public class ChatClient {
                 decoder: decoder,
                 sessionConfiguration: urlSessionConfiguration
             ),
-            { [weak self] error, completion in
+            { [weak self] completion in
                 guard let self = self else {
                     completion()
                     return
                 }
-                self.refreshToken(error: error, completion: { _ in completion() })
+                self.refreshToken(
+                    completion: { _ in completion() }
+                )
```

Comment:
@evsaev can't we just call `self.refreshToken(completion)` here? Do those 2 completions differ in their definition?